actors/evm/src/interpreter/instructions/arithmetic.rs

```diff
@@ -137,36 +134,90 @@ mod test {
             }
         }
 
+        pub fn ins_add(s: &mut Stack) {
```

Comment:
nit: we should probably just fix the tests to call the instructions directly. We only tested the instructions with a proper stack because we had no other way.

Reply:
yeah, a chore for when we have nothing better to do.